HighwayPatrol.cpp

```diff
@@ -1,18 +1,29 @@
 #include "HighwayPatrol.h"
+#include "Car.h"
+#include "HighwayPatrol.h"
+#include "Motorcycle.h"
+#include "SemiTruck.h"
 #include <iostream>
 
 HighwayPatrol::HighwayPatrol() : Vehicle("HighwayPatrol")
 {
 
 }
 
+HighwayPatrol::HighwayPatrol(const HighwayPatrol&) = default;
+HighwayPatrol& HighwayPatrol::operator = (const HighwayPatrol&) = default;
+
 void HighwayPatrol::scanHighway(Highway* h)
 {
     std::cout << name << ": scanning highway for speeders" << std::endl;
 
-    for( int i = h->vehicles.size(); --i >= 0; )
+    // Is it worth casting to silence warnings like these? 
+
+    // Explicitly casting h->vehicles.size() in order to silence the [-Wshorten-64-to-32] warning
+    for(int i = static_cast<int>(h->vehicles.size()); --i >= 0;)
     {
-        auto* v = h->vehicles[i];
+        // Explicitly casting i in order to silence the [-Wsign-conversion] warning
+        auto* v = h->vehicles[static_cast<unsigned long int>(i)];
```

Comment:
I think the updated instructions use `size_t` instead of `int`. 
you might try switching it out and seeing if you still need the `static_cast`ing